view/src/frontend/web/js/form/element/countrySelect.js

```diff
@@ -1,21 +1,21 @@
+'use strict';
+
 /**
  * @author Bartosz Herba <b.herba@ctidigital.com>
  * @copyright 2017 CtiDigital Sp. z o.o.
  */
-define([
-    'ko',
-    'Magento_Ui/js/form/element/select',
-    'GoogleAddressLookup/model/address/addressData',
-], function (ko, Element, addressData) {
+define(['ko', 'Magento_Ui/js/form/element/select', 'GoogleAddressLookup/model/address/addressData'], function (ko, Element, addressData) {
     "use strict";
 
     return Element.extend({
-        initialize: function () {
+        initialize: function initialize() {
+            var _this = this;
```

Comment:
@jonpday  a bit scary that ES6 automatically works in chrome etc
But will fail in older IE / firefox
Probably need cross browser testing for changes as a whole.